common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -185,6 +185,21 @@ class InheritanceMixin(XBlockMixin):
         default=False
     )
 
+    # TODO Find a way to include these fields without the inheritance.
```

Comment:
@cpennington @doctoryes @ormsbee Any idea how to retrieve this info without going through the inheritance structure? Loading the data from Mongo is a no-go as we have to instantiate every node...which is what lead to this method of loading the full structure.